torch/csrc/distributed/rpc/tensorpipe_agent.h

```diff
@@ -220,6 +220,16 @@ class TORCH_API TensorPipeAgent : public RpcAgent {
   size_t numPendingResponses();
   size_t messageIdToTimeoutMapSize();
 
+ protected:
+  // TensorPipe write function that could be used to write response
+  // messages by server, and write request messages by client.
```

Comment:
Let's add a comment to explain that this is under `protected` because we need to override it in FaultyTensorPipeAgent.